doc/cug.tex

```diff
@@ -4551,6 +4489,380 @@ \subsubsection{Collapsible Families In Suite Graphs}
+      # or equivalently:
+    prep => model_run1 & model_run2
+\end{lstlisting}
+and this,
+\begin{lstlisting}
+    # for run = 1, 2 and mem = cat, dog
+    graph = """prep => init<run> => model<run,mem> =>
+                    post<run,mem> => wrap<run> => done"""
+\end{lstlisting}
+expands to this,
+\begin{lstlisting}
+    prep => init_run1 => model_run1_cat => post_run1_cat => wrap_run1 => done
+            init_run1 => model_run1_dog => post_run2_dog => wrap_run1
+    prep => init_run2 => model_run2_cat => post_run2_cat => wrap_run2 => done
+            init_run2 => model_run2_dog => post_run2_dog => wrap_run2
+\end{lstlisting}
+This is visualized by \lstinline=cylc graph= in Figure~\ref{fig-params-1}.
+
+\begin{figure}
+    \begin{center}
+        \includegraphics[width=10cm]{graphics/png/orig/params1.png}
+    \end{center}
+    \caption[Parameter expansion example.]{\scriptsize
+     Parameter expansion example.}
+    \label{fig-params-1}
+\end{figure}
+
+\subsubsection{Singling Out Specific Parameter Values}
+
+Specific parameter values can be singled out in the graph and under
+\lstinline=[runtime]= with this notation: \lstinline@<param_name=param_value>@.
+For example, here's how to make a special task trigger off just the first of a
+set of model runs,
+\begin{lstlisting}
+[cylc]
+    [[parameters]]
+        run = 1..5
+[scheduling]
+    [[dependencies]]
+        graph = """model<run> => post_proc<run>  # general case
+                   model<run=1> => check_first_run  # special case"""
+[runtime]
+    [[model<run>]]
+        # config for all "model" runs...
+    [[model<run=1>
+        # special config for the first model run (if any)...
+    #...
+\end{lstlisting}
+
+\subsubsection{Parameter Offsets In The Graph}
+
+A negative offset notation \lstinline@<param_name=-1>@ is interpreted as the
+previous value in the ordered list of parameter values (integer or string).
+For example, to split a model run into multiple chunks with each chunk
+depending on the previous one, this graph:
+\begin{lstlisting}
+    graph = "model<run-1> => model<run>"  # for run = 1, 2, 3
+\end{lstlisting}
+expands to this:
+\begin{lstlisting}
+    model_run1 => model_run2 => model_run3
+\end{lstlisting}
+and this graph:
+\begin{lstlisting}
+    graph = "proc<size-1> => proc<size>"  # for size = small, big, huge
+\end{lstlisting}
+expands to this:
+\begin{lstlisting}
+    graph = "proc_small => proc_big => proc_huge"
+\end{lstlisting}
+
+\subsubsection{Task Families And Parameterization}
+
+Family names can be parameterized just like task names.  For instance, to
+define a family for tasks that share common values of a parameter,
+\begin{lstlisting}
+[runtime]
+    [[RUN<r>]]
+    [[model<r>]]
+        inherit = RUN<r>
+    [[post_proc<r>]]
+        inherit = RUN<r>
+# result: RUN_r1 contains model_r1 and post_proc_r1, and so on.
+\end{lstlisting}
+
+As described in Section~\ref{FamilyTriggers} family names can be used to
+trigger all members at once,
```

Comment:
I only picked up on this one really as its inconsistent with what follows where you are using colons in the same paragraph.